test/models/sandbox_test.rb

```diff
@@ -9,11 +9,12 @@ class SandboxTest < ActiveSupport::TestCase
   test "run a sandbox grader lazy mode" do
     skip unless ENV['TEST_SANDBOX']
 
-    assign = create(:assignment, type: "files", course: @cs101, name: "Test Assignment")
+    assign = create(:assignment, type: "Files", course: @cs101, name: "Test Assignment")
     grdtar = Rails.root.join('test', 'fixtures', 'files', 'TestScript', 'test.pl')
     upload = simulated_upload(@fred, grdtar)
     upload.save!
-    grdcfg = build(:grader, type: "SandboxGrader", upload: upload, params: "lazy.rb", assignment: assign, order: 1)
+    grdcfg = build(:grader, type: "SandboxGrader", upload: upload, params: "lazy.rb",
```

Comment:
Yea, these test are all defunct now.

Tests for sandbox graders are annoying, since they can't run on Travis and therefore break 100% of the time. That's one of the advantages to the steno plan - if sandbox grading is external, we can keep the Bottlenose repo 100% travis testable.